tsstats/tests/test_ident_map.py

```diff
@@ -24,3 +25,33 @@ def test_ident_map(identmap_clients):
     assert clients['5'] == cl
     assert clients['UID1'] == uidcl
     assert clients['UID5'] == uidcl
+
+
+@pytest.fixture(scope="module")
+def pretty_identmap():
+    return [
+        {'primary_id': '1', 'alternate_ids': ['3', '6']},
+        {'primary_id': '4', 'alternate_ids': ['9', '42']}
+    ]
+
+
+@pytest.mark.parametrize('test_input,expected', [
+    (
+        [
+            {'primary_id': '1', 'alternate_ids': ['3', '6']},
+            {'primary_id': '4', 'alternate_ids': ['9', '42', '23']}
+        ],
+        (('3', '1'), ('6', '1'), ('9', '4'), ('42', '4'), ('23', '4'))
+    ),
+    (
+        [
+            {'name': 'Friend 1', 'primary_id': '2', 'alternate_ids': ['4']},
+            {'name': 'Friend 3', 'primary_id': '8', 'alternate_ids': ['9', '14']}
+        ],
+        (('4', '2'), ('9', '8'), ('14', '8'))
+    )
+])
+def test_flip_pretty_identmap(test_input, expected):
```

Comment:
No problem :) Thanks for keeping up the great work on this great contribution despite all my change-requests!